PlayList/src/com/playlist/PlayerState.java

```diff
@@ -0,0 +1,7 @@
+package com.playlist;
+
+public interface PlayerState {
+	
+	public void execute(AudioPlayer aPlayer);
```

Comment:
[High] Intead of having just one method (that then you need to use a switch to check the action), you should have all possible actions as methods